tests/test_cloudformation/test_cloudformation_stack_crud_boto3.py

```diff
@@ -2309,3 +2319,21 @@ def get_role_name():
                 AssumeRolePolicyDocument="some policy",
                 Path="/my-path/",
             )["Role"]["Arn"]
+
+
+@mock_cloudformation()
+def test_update_changeset_yaml_and_json():
```

Comment:
Can we re-use an existing test to verify this is fixed? For example, the `test_describe_change_set` already verifies the  creating/describing/executing of a change set. It should be trivial to parametrize that test instead, and verify the entire flow with either JSON or YAML